server/src/routes/api/v1/moviesRouter.js

```diff
@@ -17,9 +22,15 @@ moviesRouter.get("/:id", async (req, res) => {
     try {
         const movie = await Movie.query().findById(id)
         const reviews = await movie.$relatedQuery("reviews")
-        movie.reviews = reviews
+        const serializedReviews = await Promise.all(reviews.map(async review => {
```

Comment:
if this route is serializing its associated Reviews, the Movie should also be serialized at some point